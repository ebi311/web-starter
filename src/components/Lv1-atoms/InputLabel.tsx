import React from 'react';
import { styled } from '../Theme';

const Label = styled.label`
  font-weight: bold;
  font-size: ${p => p.theme.fontSize * 0.8}px;
`;

const InputLabel: React.FC = props => {
  return <Label>{props.children}</Label>;
};

export default InputLabel;
