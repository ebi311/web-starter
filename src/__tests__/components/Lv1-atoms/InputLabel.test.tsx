import { shallow } from 'enzyme';
import React from 'react';
import InputLabel from '../../../components/Lv1-atoms/InputLabel';

describe('InputLabel', () => {
  test('success', () => {
    const wrapper = shallow(
      <InputLabel>
        <div>hoge</div>
      </InputLabel>,
    ).dive();
    expect(wrapper).toMatchSnapshot();
  });
});
