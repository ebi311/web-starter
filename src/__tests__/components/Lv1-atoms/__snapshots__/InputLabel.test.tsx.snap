// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputLabel success 1`] = `
<label
  className="sc-AxjAm ipInQF"
>
  <div>
    hoge
  </div>
</label>
`;
